department-of-veterans-affairs/caseflow
# Daily docket UI

Connects #7073 

### Description
This PR adds the UI for the Daily Docket page. The information is currently stubbed out and not connected to Redux. Those updates will come in future PRs.

### Testing Plan
1. Go to /hearings/schedule/docket/RO04/2018-10-02
1. Confirm UI looks correct
